com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/ApplicationConfiguration.java

```diff
@@ -755,9 +755,9 @@ private Integer getHealthCheckTimeRangeFromEnvironment() {
 
     private String getApplicationGuidFromEnvironment() {
         Map<String, Object> vcapApplicationMap = getVcapApplication();
-        String applicationGuid = (String) vcapApplicationMap.get("application_id");

```

Comment:
This change was doe to a sonar code smell because there is an instance variable named `applicationGuid` and this was the same, although with a smaller scope